frontend/cypress/integration/search.e2e.js

```diff
@@ -20,37 +20,29 @@ context('Searchbar', () => {
     // Arrange
     /* eslint-disable no-unused-expressions -- expect(..).to.Be.Called return can be ignored */
     const spySearch = cy.spy((req) => req.continue());
-    const spyGetNodesById = cy.spy((req) => req.continue());
 
     cy.intercept(`${apiBaseUrl}/search/all*`, spySearch).as('searchQuery');
-    cy.intercept(`${apiBaseUrl}/getNodesById*`, spyGetNodesById).as(
-      'getNodesQuery'
-    );
 
     // Act 1
     cy.get('.SearchBar').type('keanu');
 
     // Assert 1
-    cy.wait(['@searchQuery', '@getNodesQuery']).then(() => {
+    cy.wait(['@searchQuery']).then(() => {
```

Comment:
why did you remove all that extra tests?

Reply:
Because this is no longer called any longer. This was the reason to even do these changes.